Source/AudibleUtilities/Widevine/Cdm.Api.cs

```diff
@@ -0,0 +1,189 @@
+﻿using AudibleApi;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Net.Http;
+using AudibleApi.Cryptography;
+using Newtonsoft.Json.Linq;
+using Dinah.Core.Net.Http;
+using System.Text.Json.Nodes;
+
+#nullable enable
+namespace AudibleUtilities.Widevine;
+
+public partial class Cdm
```

Comment:
This is my C# implementation of the https://github.com/devine-dl/pywidevine project.